providers/mondoo/resources/mondoo_space.go

```diff
@@ -39,7 +57,7 @@ func (m *mqlMondooSpace) assets() ([]any, error) {
 		} `graphql:"assets(spaceMrn: $spaceMrn)"`
 	}
 	vars := map[string]any{
-		"spaceMrn": mondoogql.String(conn.Upstream.SpaceMrn),
+		"spaceMrn": mondoogql.String(m.Mrn.Data),
```

Comment:
this means we use the mql MRN to fetch data. this means if we connect via a mondoo org this still works